content/manuals/copilot/usage.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Using the Docker for GitHub Copilot extension
+linkTitle: Usage
+description: |
+  Learn how to use the Docker for GitHub Copilot extension to interact with the
+  Docker agent, get help Dockerizing projects, and ask Docker-related questions
+  directly from your IDE or GitHub.com.
+weight: 20
+---
+
+{{% restricted title="Early Access" %}}
+The Docker for GitHub Copilot extension is an [early access](/release-lifecycle#early-access-ea) product.
+{{% /restricted %}}
+
+The Docker Extension for GitHub Copilot provides a chat interface that you can
+use to interact with the Docker agent. You can ask questions and get help
+Dockerizing your project.
+
+The Docker agent is trained to understand Docker-related questions, and provide
+guidance on Dockerfiles, Docker Compose files, and other Docker assets.
+
+## Setup
+
+Before you can start interacting with the Docker agent, make sure you've
+[installed](./install.md) the extension for your organization.
```

Comment:
maybe we should say user rather than organization here, then provide a separate section or note on organizations